src/lib/util/get-prop-from-string.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Get the value of an object's (nested) property from a string.
+ *
+ * This function does not do any processing or validation of the property path.
+ * Empty paths and non-existent properties yield `undefined`. Whitespace is
+ * valid as a property name (so `x. .y`) is a valid path.
+ *
+ * @param {string} propertyPath The path to the property you want to extract. First part of the path is assumed to be on the root level of the `object` you provide.
+ * @param {object} object The object you want to extract the property from.
+ * @return {T = unknown | undefined} If the nested property exists, return that as `unknown` or try to coerce it into to the type you've provided. Otherwise, return `undefined`.
+ */
+export const getPropFromString = <T = unknown>(
```

Comment:
I'd suggest we use https://lodash.com/docs/4.17.15#get that is exposed as micro lib here: https://www.npmjs.com/package/lodash.get

Reply:
Interesting. I'd be on board with doing that, but adding an external dependency for that feels a little ... unnecessary? But happy to do it if you think it's worth the tradeoff. 

Out of curiosity: what's your reason for preferring an external lib to writing the function ourself?

Reply:
* This dependency is deliberately a single function not the entire lib so it doesn't bring anything extra we don't need (I'm against bringing code we don't need)
* non-differentiator, just a generic boring fn I'd rather not review, maintain, look at the code no matter how well written it is
* We don't have to maintain tests for it, lodash does

Reply:
Yeah, I buy that. Done in e002678050454a967aa4a3164a5e10a83ebc7268.